### PR TITLE
Freshdesk sdk demo app

### PR DESCRIPTION
**Pull request by ravirajsubramanian**

### CODE REVIEW

**Review by Adithya-Natarajan-T18333 — APPROVED** [review]

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    ├── app                        Contains the files that are required for the front end component of the app
+    │   ├── app.js                 JS to render the dynamic portions of the app
+    │   ├── icon.svg               Sidebar icon SVG file. Should have a resolution of 64x64px.
+    │   ├── freshdesk_logo.png     The Freshdesk logo that is displayed in the app
+    │   ├── style.css              Style sheet for the app
+    │   ├── template.html          Contains the HTML required for the app’s UI
+    ├── config                     Contains the installation parameters and OAuth configuration
+    │   ├── iparams.json           Contains the parameters that will be collected during installation
+    │   └── iparam_test_data.json  Contains sample Iparam values that will used during testing
+    └── manifest.json              Contains app meta data and configuration information
```
Comment (Saif-Shines) [inline Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/README.md:18]: Hopefully you'd rewrite this according [to the template](https://github.com/freshworks/marketplace-sample-apps/blob/master/.github/PULL_REQUEST_TEMPLATE/APP_README_TEMPLATE.md).

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/app.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+$(document).ready(function () {
+    app.initialized()
+        .then(function (_client) {
+            window.client = _client;
+            client.events.on('app.activated',
+                function () {
+
+                });
+        });
+});
```
Comment (Saif-Shines) [inline Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/app.js:103]:
```
function onDocumentReady() {
    app.initialized()
        .then(function (_client) {
            window.client = _client;
            socketIo();
            client.events.on('app.activated', onAppActivated);
        }).catch(function (error) {
            console.error('The app failed to get initialized');
            console.error(error);
        });
};

$(document).ready(onDocumentReady);
```

Above is snippet taken from sample cti app. Is possible to make this piece of code as readable as this?

_This wouldn't be blocker to approve this PR_

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/server/server.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  searchContact: function (args) {
+    const fd = new Freshdesk({ domain: args.iparams.domain, api_key: args.iparams.api_key });
+    const query = `"email:'${args.query}'"`;
+
+    /**
+     * TASK: 4
+     *
+     * Add handler code to search contact with SDK method
+     */
```
Comment (Saif-Shines) [inline Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/server/server.js:93]: Will this code also be added?

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/template.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    <br><br>
+    <fw-input id="contactEmail" label="Contact Email"
+      placeholder="Enter the contact email to search" state="normal"
+      clear-input>
+    </fw-input>
+    <!--
+      TASK: 3
+
+      Add a Crayons button to call searchContact() method upon click action.
+    -->
```
Comment (Saif-Shines) [inline Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/template.html:40]: Will this task's code be added?

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/README.md (hunk 1 above)
Comment (Saif-Shines) [reply Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/README.md:18]: Would be good idea to link to glitch as well somewhere in the readme?

Comment (ravirajsubramanian) [reply Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/README.md]: Modified the readme according to the template and added Glitch link as well.

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/app.js (hunk 2 above)
Comment (ravirajsubramanian) [reply Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/app.js]: Done. Thanks.

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/template.html (hunk 4 above)
Comment (ravirajsubramanian) [reply Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/app/template.html]: Added the relevant code.

File: Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/server/server.js (hunk 3 above)
Comment (ravirajsubramanian) [reply Play-Along/Freshworks-Huddle/August-2020/freshdesk-sdk/server/server.js]: Added the relevant code.

**Review by Saif-Shines — APPROVED** [review]

**Review by Adithya-Natarajan-T18333 — APPROVED** [review]

**Pull request merged**